css/style.css

```diff
@@ -149,3 +149,555 @@ section .content p {
     margin-left: 45px;
   }
 }
+
+.section-1 .first-header h2 {
+  position: static;
+  width: 327px;
+  height: 97px;
+  font-size: 40px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #172b4d;
+  margin-top: 60rem;
+  margin-left: 2.1rem;
+  line-height: 52px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.section-1 .first-header span {
+  display: flex;
+  justify-content: center;
+  width: 10%;
+  height: auto;
+  border-bottom: 4px solid #36b37e;
+  border-radius: 5px;
+  text-align: center;
+  margin-left: 45%;
+}
+
+.section-1 .first-project {
+  position: static;
+  left: 0;
+  right: 0.6%;
+  top: 0.13%;
+  bottom: 83.92%;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.section-1 .first-project .dsc-project .project-header {
+  position: static;
+  width: 295px;
+  height: 57px;
+  left: 16px;
+  top: 16px;
+  padding-left: 1rem;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #3a4a42;
+}
+
+ul {
+  text-decoration: none;
+  list-style: none;
+}
+
+.section-1 > section {
+  display: grid;
+  grid-gap: 10px;
+  background-color: #2196f3;
+  padding: 10px;
+}
```

Comment:
- [ ] As per project requirements elements in the "Works" section you should use CSS Grid and Flexbox combined. Kindly correct the use of the grid for the work section. 
Hint: There is no section inside `.section-1` class.

Reply:
Hi,
![section-1](https://user-images.githubusercontent.com/74506933/125822291-468ca1ff-3313-43be-83df-634afd4c8d39.PNG)
 @hemant-soni-vst-au4 thank you for reviewing my project. I wanted to ask you something about CSS Grid because I have only one chance to submit so I needed to make sure. Can you please tell me if I did it right ? I will send you a copy of the  .section-1 class. Thank you